### ISSUE TITLE
trained weights

### ISSUE THREAD

**Issue by haveeshg** [aayushmnit/Deep_learning_explorations#7]
hi, can you provide the trained weights for this model please? If it is already there somewhere, can you point me to the link?

**Comment by tomilles** [comment]
yes please, unfortunately it takes ages on my laptop to train this model ... if you have a pretrained model or weights can you share it please?

**Comment by tomilles** [comment]
for the 7. Facial attribute section
